### ISSUE TITLE
Auto buff during combat

### ISSUE THREAD

**Issue by kolroser** [AznamirWoW/PallyPower#25]
Hello,
is there a way you could implement an option for the auto buff to also work during combat?
With SOD and the short fights, tanks are constantly pulling and it's hard to keep up the short buffs since the whole group is out of combat only a few times during a dungeon or a raid.

**Comment by AznamirWoW** [comment]
Due to limitations imposed by Blizzard on the addon automation, there's no way for any addon to automatically select a target and cast a spell in combat. You can always manually mouse over and click on a player frame requiring a buff.

**Comment by AznamirWoW** [comment]
I meant clicking on the player buttons of PallyPower.

https://imgur.com/ctSWhbY.png

**Comment by AznamirWoW** [comment]
Same reason - you have to manually target and cast in combat.

**Comment by AznamirWoW** [comment]
No, because a keypress can not cycle thru different targets automatically.

**Comment by AznamirWoW** [comment]
let me explain

when you press a button, it can trigger a spell cast
for a successful spell cast in combat the target and spell has to be set before the combat starts and once it starts it can not be changed.

different mouse over cast mods set only one spell and 'mouse over target' as a target, so the player can just wave the mouse cursor over and press one button to cast the same spell over and over.

for PallyPower it is expected that there may be different spells assigned to the same paladin for different classes, so the addon would need to look up the spell assigned for a target, set the values on the button pre-action (which it can not do in combat) and only then the spell may be cast. So you're asking for something that can only be done half-assed = assuming that a paladin can only be assigned one spell to cast.

**Comment by AznamirWoW** [comment]
So in short you may be better off just making a macro and assigning a keybing to it

/cast [@mouseover,help,nodead][] Blessing of Might
